include/z64.h

```diff
@@ -1404,10 +1404,10 @@ typedef struct {
 typedef struct {
     /* 0x00 */ u32 unk_00;
     /* 0x04 */ u32 setScissor;
-    /* 0x08 */ Color_RGBA8 primColor;
-    /* 0x0C */ Color_RGBA8 envColor;
+    /* 0x08 */ Color_RGBA8_u32 primColor;
+    /* 0x0C */ Color_RGBA8_u32 envColor;
     /* 0x10 */ u16* tlut;
-    /* 0x14 */ Gfx* monoDl;
+    /* 0x14 */ Gfx* monoDL;
```

Comment:
```suggestion
    /* 0x14 */ Gfx* dList;
```
I think `mono` is implied from the overall struct.